backend/private-graph/graph/schema.resolvers.go

```diff
@@ -826,33 +866,6 @@ func (r *mutationResolver) DeleteProject(ctx context.Context, id int) (*bool, er
 	return &model.T, nil
 }
 
-// SendAdminProjectInvite is the resolver for the sendAdminProjectInvite field.
-func (r *mutationResolver) SendAdminProjectInvite(ctx context.Context, projectID int, email string, baseURL string) (*string, error) {
```

Comment:
This method didn't appear to be used. This threw me because I thought you could be added to a project OR a workspace, but it looks like we only allow joining a workspace.